src/nn/ops/conv2d.ts

```diff
@@ -191,16 +191,10 @@ export class Conv2d extends SingleOutputOperation implements FusedOperation {
       fused = true;
     } else if (
         this.groups_ === inputChannels && this.groups_ === filter.shape[2]) {
-      if (padding === 'valid' || padding === 'same' ||
-          (padding instanceof Array && padding[1][0] === padding[1][1] &&
+      if ((padding instanceof Array && padding[1][0] === padding[1][1] &&
             padding[1][0] === padding[2][0] &&
             padding[1][0] === padding[2][1])) {
-        let fusedDepthwisePad: 'valid'|'same'|number;
-        if (padding === 'valid' || padding === 'same') {
-          fusedDepthwisePad = padding;
-        } else {
-          fusedDepthwisePad = padding[1][0];
-        }
+        const fusedDepthwisePad: 'valid'|'same'|number = padding[1][0];
```

Comment:
`fusedDepthwisePad` will never be `same` or `valid` right?
And why don't we keep using `ExplicitPadding`?

Reply:
@miaobin Yes, here `fusedDepthwisePad` will never be `same` or `valid`.
Since `pad` of `tf.fused.depthwiseConv2d()` supports [`'valid'|'same'|number`](https://github.com/tensorflow/tfjs/blob/master/tfjs-core/src/ops/fused/depthwise_conv2d.ts#L106), does not support `ExplicitPadding`, so here we could set fusedDepthwisePad as number.